client/src/styles/fonts.module.scss

```diff
@@ -0,0 +1,74 @@
+// Font for headers etc. is:'Sofia Sans'
+@import url('https://fonts.googleapis.com/css2?family=Sofia+Sans:ital,wght@0,700;0,800;0,900;1,700;1,800;1,900&display=swap');
+// Font for text blocks etc. is: 'Lato'
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;1,300;1,700&display=swap');
+
+$header-font: 'Sofia Sans', sans-serif;
+$body-font: 'Lato', 'Calibri', sans-serif;
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: $header-font;
+}
+
+* {
+  font-family: $body-font;
+  font-size: 16px;
```

Comment:
You could use rem values for font-sizes (1rem being 16px normally -> 0,125rem = 2px). This makes it more accessible (since you can adjust the base font-size in the browser as a user)